ownership-chain/runtime/src/precompiles/test.rs

```diff
@@ -27,4 +27,6 @@ fn ethereum_precompiled_addresses_are_precompile() {
 	assert!(is_precompile(hash(1027)).unwrap());
 	assert!(is_precompile(H160::from_str("0xffffffffffffffffffffffff0000000000000005").unwrap())
 		.unwrap());
+	assert!(is_precompile(H160::from_str("0x0000000000000000000000010000000000000005").unwrap())
```

Comment:
why use `0x000000000000000000000001` instead of `0xfffffffffffffffffffffffe` for the first part of the contract ?

Reply:
because it was required the new contract addresses to be different as the existing ones